Jenkinsfile

```diff
@@ -46,31 +49,9 @@ pipeline {
           }
           axis {
             name 'PLATFORM'
-            values 'linux/amd64', 'linux/arm64'
+            values 'amd64', 'arm64'
```

Comment:
ARCH?
platform is "linux/amd64", where "amd64" is docker platform arch.

Reply:
We have ARCH as a separate axis, I removed the "linux" as the platform provided a convenient tag name, I can revert this and do some string substitution at the site of use, which is probably more clear